Product/Production/Common/CONNECTCommonWeb/src/main/java/gov/hhs/fha/nhinc/connectmgr/uddi/UDDIUpdateManagerHelper.java

```diff
@@ -52,126 +41,31 @@
 public class UDDIUpdateManagerHelper {
 
     private static final Logger LOG = LoggerFactory.getLogger(UDDIUpdateManagerHelper.class);
-    private static final String GATEWAY_PROPERTY_FILE = "gateway";
-    private static final String UDDI_REFRESH_KEEP_BACKUPS_PROPERTY = "UDDIRefreshKeepBackups";
-    private static final String UDDI_MAX_NUM_BACKUPS_PROPERTY = "UDDIMaxNumBackups";
-    private static final String UDDI_UPDATE_TIME = "UDDILastUpdate";
-    private static final int MAX_NUM_BACKUP = 10;
-    private static final ArrayList<String> backupFileList = new ArrayList<>();
 
     /**
      * This method retrieves the BusinessDetail data from the UDDI server. The BusinessDetail contains the list of
      * BusinessEntity that contains all the necessary information for each services.
      *
      * @return The BusinessDetail data retrieved from UDDI.
      */
-    private static BusinessDetail retrieveDataFromUDDI() throws UDDIAccessorException {
+    private static BusinessDetail retrieveDataFromUDDI(String targetURL) throws UDDIAccessorException {
         UDDIAccessor oUDDIAccessor = new UDDIAccessor();
-        return oUDDIAccessor.retrieveFromUDDIServer();
+        return oUDDIAccessor.retrieveFromUDDIServer(targetURL);
     }
 
-    /**
-     * This method is called to force a refresh of the uddiConnectionInfo.xml file. It will retrieve the data from the
-     * UDDI server and update the local XML file.
-     *
-     * @throws UDDIAccessorException
-     */
-    public void forceRefreshUDDIFile() throws UDDIAccessorException {
+    public BusinessDetail forceRefreshUDDIFile(String targetURL) throws UDDIAccessorException {
         if (LOG.isDebugEnabled()) {
             LOG.debug("Start: UDDIUpdateManagerHelper.forceRefreshUDDIFile method - loading from UDDI server.");
         }
 
         try {
-            BusinessDetail businessDetail = retrieveDataFromUDDI();
-            if (businessDetail != null) {
-                createUddiFileBackupByRenaming();
-                saveUddiResultsToFile(businessDetail);
-            }
+            return retrieveDataFromUDDI(targetURL);
 
-        } catch (Exception e) {
+        } catch (UDDIAccessorException e) {
             String sErrorMessage = "Failed to retrieve data from UDDI.  Error: " + e.getLocalizedMessage();
             LOG.error("Error refreshing UDDI: {}", sErrorMessage, e);
```

Comment:
should this merged into one string?

LOG.error("Error refreshing UDDI: Failed to retrieve data from UDDI.  Error: {}", e.getLocalizedMessage(), e);